spotbugs/src/main/java/edu/umd/cs/findbugs/Project.java

```diff
@@ -237,18 +237,26 @@ public boolean addFile(String fileName) {
     }
 
     /**
-     * Add a source directory to the project.
+     * Add source directories to the project.
      *
-     * @param dirName
-     *            the directory to add
-     * @return true if the source directory was added, or false if the source
-     *         directory was already present
+     * @param dirNames
+     *            the directories to add
```

Comment:
I hate such API.

The old javadoc was fuzzy - what is a "directoryName"? Is this just a name? a relative path? an absolute path? 
Looking at the FindBugsWorker we see that this is full absolute path to the directory, NOT a name. Looking into the makeAbsoluteCwdCandidates() we see it can be almost anything.

So obviously the javadoc is lying here. It means: either full absolute paths or paths relative to any of the "working" directories added via addWorkingDir().

Please fix this.